models/employee.py

```diff
@@ -79,11 +79,25 @@ def update_from_dict(self, d):
         self.photo_url = d.get('photo_url')
         self.department = d.get('department')
         self.meta_department = get_meta_department(self.department)
+        self.terminated = False
+
+    def is_employee_data_changed(self, d, is_terminated):
+        result = any(
+            [
+                self.first_name != d.get('first_name'),
+                self.last_name != d.get('last_name'),
+                self.photo_url != d.get('photo_url'),
+                self.department != d.get('department'),
+                self.meta_department != get_meta_department(self.department),
+                self.terminated is not is_terminated,
+            ]
+        )
+        return result
 
     def get_gravatar(self):
         """Creates gravatar URL from email address."""
         m = hashlib.md5()
-        m.update(self.user.email())
+        m.update((self.username + '@' + config.DOMAIN).encode())
```

Comment:
You could make `email` a property, like `full_name` is already.

Reply:
good idea, will do.